<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTestWidget</class>
 <widget class="QWidget" name="AddTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>测试类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>手机平台</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>选择工具</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_test">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_platform">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_tool">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start_test">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>开始测试</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
